mmpose/datasets/datasets/base/kpt_2d_sview_rgb_img_top_down_dataset.py

```diff
@@ -60,6 +60,8 @@ def __init__(self,
         self.ann_info['num_output_channels'] = data_cfg['num_output_channels']
         self.ann_info['dataset_channel'] = data_cfg['dataset_channel']
 
+        self.ann_info['use_different_joint_weights'] = False
```

Comment:
How about 
`self.ann_info['use_different_joint_weights'] = data_cfg.get('use_different_joint_weights', False)`

Reply:
This seems better, I will change it.